java20181113numbers/src/main/java/orfiel/Main42.java

```diff
@@ -0,0 +1,33 @@
+package orfiel;
+
+//import klasy Scanner
+import java.util.Scanner;
+
+public class Main42 {
+
+    public static void main(String[] arg) {
+
+        //deklaracja zmiennej "liczba" i przypisanie jej wartości
+        int liczba = 0;
+
+        //dodanie obiektu "sc" odbierającego dane wprowadzone przez urzytkownika
+        Scanner sc = new Scanner(System.in);
+
+        //uruchomienie pętli
+        while (liczba !=42) {
+
+            //prośba o liczbę
+            System.out.println("Podaj liczbę całkowitą");
+
+            //nadpisanie wartości "liczba" przez wartość wprowadzoną przez urzytkownika
+            liczba = sc.nextInt();
+
+            //wypisanie warości podanej przez urzytkownika
+            System.out.println(liczba);
+
+        //warunek zakończenia działania pętli i zamknię programu gdy pojawi się liczba 42
+        }
+    }
+}  //jak zrobić żeby ta kurwa na końcu się nie wyświetlała i czemu
```

Comment:
jesli chesz zeby sie nei wyświetlała 42 to musisz zmienic troche pętle.
While true powoduje ze bedzie to nieskończona pętla. 
If z 42 i breake oznacza ze Jak bedzie 42 to wyjdziemy z pętli
`while (true) { 
            System.out.println("Podaj liczbę całkowitą");
            liczba = sc.nextInt();
            if (liczba==42) {
break;
}
            System.out.println(liczba);
        }`